ui/gulpfile.esm.js

```diff
@@ -28,11 +28,40 @@ task(
   shell.task('yarn react-app-rewired build --library')
 )
 
-task('build', series('swagger:generate', 'webpack:build'))
+task(
+  'supportedBrowsers',
+  shell.task(
+    'echo "window.__supported_browsers__ = $(browserslist-useragent-regexp --allowHigherVersions)" > ./public/supportedBrowsers.js'
+  )
+)
+
+task(
+  'unSupportedBrowsers',
+  shell.task(
+    'echo "window.__unsupported_browsers__ = $(browserslist-useragent-regexp "dead, IE 11" --allowHigherVersions)" > ./public/unSupportedBrowsers.js'
+  )
+)
```

Comment:
The generated full list is here: https://browserl.ist/?q=dead%2C+IE+11

Reply:
It won't work in some old Chrome versions as well. The list needs some tuning..